docs/reuse/tutorial/setup.rst

```diff
@@ -1,34 +1,46 @@
 We recommend starting from a clean Ubuntu 22.04 installation. If you don't have
 one available, you can create one using `Multipass`_:
 
-.. collapse:: How to create an Ubuntu 22.04 VM with Multipass
+.. tabs::
 
-    Is Multipass_ already installed and active? Check by running
+    .. tab:: Ubuntu
 
-    .. code-block:: bash
+        Is Multipass_ already installed and active? Check by running
 
-        sudo snap services multipass
+        .. code-block:: bash
 
-    If you see the ``multipass`` service but it isn't "active", then you'll
-    need to run ``sudo snap start multipass``. On the other hand, if you get
-    an error saying ``snap "multipass" not found``, then you must install
-    `Multipass <install-multipass_>`_:
+            sudo snap services multipass
 
-    .. code-block:: bash
+        If you see the ``multipass`` service but it isn't "active", then you'll
+        need to run ``sudo snap start multipass``. On the other hand, if you get
+        an error saying ``snap "multipass" not found``, then you must install
+        `Multipass <install-multipass_>`_:
 
-        sudo snap install multipass
+        .. code-block:: bash
 
-    Then you can create the VM with the following command:
+            sudo snap install multipass
 
-    .. code-block:: text
+    .. tab:: Windows
```

Comment:
The tabs are nice! Out of scope for this PR, but it would be neat if we could select the relevant tab "onload" based on the user agent. This could also work for the multipass docs.

Reply:
yeh I couldn't find a way to have the link switch correctly to the OS